source/options.tsx

```diff
@@ -66,14 +66,16 @@ async function init(): Promise<void> {
 			</select>
 		) as unknown as HTMLSelectElement;
 		form.before(<p>Domain selector: {dropdown}</p>, <hr/>);
-		dropdown.addEventListener('change', () => {
+		dropdown.addEventListener('change', (event) => {
```

Comment:
Is the linter on GitHub Actions not working anymore? I think this should be without parens

Reply:
CI isn't running on forks. I'll double check it.

Reply:
Ah yes, it’s only set to run on push but not on PRs.

Can you make that change too? And then it can be merged if it runs on this PR